src/main/g8/.gitattributes

```diff
@@ -0,0 +1,18 @@
+$if(false)$
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+$endif$
+
+/src/test/resources/**/data/** text=false
```

Comment:
You might want to remove the blank line at 17 or move it before line 16 so that the generated .gitattributes doesn't start with a blank line.  Also, should the directory at 18 start with a slash - shouldn't it be a relative path (`src/test/resources/**/data/**`)?

Reply:
Yep, I'll fix all the blank line issues.

I think the beginning slash actually doesn't make a difference in this pattern, but it does look at bit weird. I'll remove it.